docs/devsguide/new_dev_guide.rst

```diff
@@ -92,15 +35,11 @@ developers, it is recommended that you install PyNE from the source.
 
 Make sure that your device has all of the dependencies downloaded. You 
 can check this by typing the name of the program into your command line 
-followed by "-v" (e.g. $ python -v). If you are just starting as a 
+followed by "--version" (e.g., $ python --version). If you are starting as a
```

Comment:
From the other PR, I had suggested to change $ python --version to have the double ` so that it renders as code.

Reply:
That's really strange, @opotowsky I thought we fixed this in my PR by changing the line to 
```suggestion
followed by :code:`--version` (e.g., :code:`$ python --version`). If you are starting as a
```

Reply:
maybe I messed up :(

it is in now